terraform/modules/public-load-balancer/security_groups.tf

```diff
@@ -2,6 +2,13 @@ resource "aws_security_group" "public_alb" {
   name        = "fargate_${var.app_name}_${var.workspace}_public_alb"
   vpc_id      = var.vpc_id
   description = "${var.app_name} Internet-facing ALB in govuk-${var.workspace} cluster"
+  
+  tags = merge(
+    var.additional_tags,
+    {
+      name      = "${var.app_name}-sg"
```

Comment:
it doesn't have the workspace interpolated in comparison to the previous tags above and name of the resource too